algo/src/test/kotlin/com/hartwig/actin/algo/evaluation/treatment/HasRadiologicalProgressionFollowingLatestTreatmentLineTest.kt

```diff
@@ -0,0 +1,177 @@
+package com.hartwig.actin.algo.evaluation.treatment
+
+import com.hartwig.actin.algo.evaluation.EvaluationAssert.assertEvaluation
+import com.hartwig.actin.datamodel.algo.EvaluationResult
+import com.hartwig.actin.datamodel.clinical.TreatmentTestFactory
+import com.hartwig.actin.datamodel.clinical.treatment.history.StopReason
+import com.hartwig.actin.datamodel.clinical.treatment.history.TreatmentResponse
+import org.assertj.core.api.Assertions.assertThat
+
+
+import org.junit.Test
+
+class HasRadiologicalProgressionFollowingLatestTreatmentLineTest {
+
+    @Test
+    fun shouldFailWhenTreatmentHistoryEmpty() {
```

Comment:
We take advantage of the kotlin feature to backtick quote test names eg

``fun `Should fail when treatment history is empty`()``